core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -227,100 +226,76 @@
               (mouseleave)="closeSubmenuIfNotMobile($event)">
             <div class="d-flex flex-column">
               <div class="top-section d-flex">
-                <a class="item item-border d-flex flex-column"
-                   (keydown)="onMenuKeypress($event, 'getInvolvedMenu', {shiftTab: ACTION_CLOSE, enter: ACTION_CLOSE})"
-                   href="/partnerships"
-                   [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.PARTNERSHIPS.ROUTE"
-                   rel="noopener">
+                <div class="item item-border d-flex flex-column innvolved-item" onclick="location.href='/partnerships'"
+                     (keydown)="onMenuKeypress($event, 'getInvolvedMenu', {shiftTab: ACTION_CLOSE, enter: ACTION_CLOSE})">
                   <div class="heading">
-                    <span class="material-icons language">language</span>
-                    {{'I18N_TOPNAV_PARTNERSHIPS' | translate}}
+                    <span class="material-icons language mx-1">language</span>
+                    <a class="heading-text underline" href="/partnerships"
+                       [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.PARTNERSHIPS.ROUTE" rel="noopener">{{'I18N_TOPNAV_PARTNERSHIPS' | translate}}
+                    </a>
                   </div>
                   <div class="des">
                     {{'I18N_TOPNAV_PARTNERSHIPS_DESCRIPTION' | translate}}
                   </div>
-                </a>
-                <a class="item d-flex flex-column"
-                   href="/volunteer"
-                   [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.VOLUNTEER.ROUTE"
-                   (keydown)="onMenuKeypress($event, 'getInvolvedMenu', {enter: ACTION_CLOSE})"
-                   rel="noopener">
+                </div>
+                <div class="item d-flex flex-column innvolved-item" onclick="location.href='/volunteer'">
                   <div class="heading">
-                    <span class="material-icons volunteer">supervisor_account</span>
-                    {{'I18N_HEADING_VOLUNTEER'| translate}}
+                    <span class="material-icons volunteer mx-1">supervisor_account</span>
+                    <a class="heading-text underline" href="/volunteer" [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.VOLUNTEER.ROUTE"
+                       (keydown)="onMenuKeypress($event, 'getInvolvedMenu', {enter: ACTION_CLOSE})"
+                       rel="noopener">{{'I18N_HEADING_VOLUNTEER'| translate}}
+                    </a>
```

Comment:
```suggestion
                    <a class="heading-text underline"
                       href="/volunteer"
                       [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.VOLUNTEER.ROUTE"
                       (keydown)="onMenuKeypress($event, 'getInvolvedMenu', {enter: ACTION_CLOSE})"
                       rel="noopener">
                         {{'I18N_HEADING_VOLUNTEER'| translate}}
                    </a>
```
Ditto at all other places.

Reply:
Done